docs/topics/modules/analytics_stack_reports.adoc

```diff
@@ -0,0 +1,31 @@
+[id="analytics_stack_reports"]
+= Analytics stack reports
+
+The {osio} analytics stack report provides comprehensive information regarding your  application stack and its component libraries that helps you make informed choices. {osio} analyzes your stack and recommends alternate and additional components to improve your application.
+
+The stack report is available after the first pipeline build completes. To view the stack report for your quickstart application:
+
+. Return to your space dashboard in the {osio} browser tab. The *Stack Report Recommendations* panel in the dashboard shows a summary of the stack report.
++
+image::stack_reports.png[Stack Reports]
++
+. Click *View complete stack report* to view the detailed stack report. The detailed stack report provides information about *Security Issues*, *Licenses*, *Insights*, and *Dependency Details* for your stack.
++
+image::detailed_stackreport.png[Detailed Stack Report]
++
+. Click each of the cards to view the following detailed information:
++
+* *Security Issues* lists dependencies with known security vulnerabilities and the relevant Common Vulnerabilities and Exposures (CVE) information.
+* *Licenses* lists dependencies with license conflicts and unknown licenses and suggests alternate dependencies to resolve the conflicts.
+* *Insights* identifies dependencies that do not work well with the stack and recommends alternate dependencies for them. In addition it recommends companion dependencies to enhance your stack. For this example, the stack report recommends companion dependencies that augment your development stack.
+* *Dependency Details* lists all the  dependencies in your stack, flags security, usage, and license issues for them, and suggests replacements for dependencies with issues.
++
+For further information about the stack report see link:user-guide.html#interpreting_stack_report[interpreting your stack report].
+
+. In the *Companion Dependency Details* section, click *Create work item* to add an auto-populated work item to the planner.
+
+You have now learned how stack report provides detailed information about your stack and its dependencies, suggests alternate and additional dependencies you can add to your stack to improve your application and make it more secure.
+
+Now that you have completed the *Use analytics to identify and resolve security and license issues in the application* task in the *Test Iteration*, ensure that you change the state of the work item and its parent feature to *Closed* using the *Plan* tab.
```

Comment:
Are this and the next para really steps? i.e. after doing the above, do <this para> and then <next para> to complete this task?